js/script.js

```diff
@@ -83,29 +115,37 @@
 
 		if (typeof cpuLoadChart === 'undefined') {
 			cpuLoadChart = new SmoothieChart(
-			{
-				millisPerPixel: 100,
-				minValue: 0,
-				grid: { fillStyle: 'rgba(249,249,249,1)', strokeStyle: 'transparent' },
-				labels: { fillStyle: 'rgba(0,0,0,0.4)', fontSize: 12 }
-			});
+				{
+					millisPerPixel: 100,
+					minValue: 0,
+					grid: {fillStyle: 'rgba(249,249,249,1)', strokeStyle: 'transparent'},
+					labels: {fillStyle: 'rgba(0,0,0,0.4)', fontSize: 12},
+					responsive: true
+				});
 			cpuLoadChart.streamTo(document.getElementById("cpuloadcanvas"), 1000/*delay*/);
 			cpuLoadLine = new TimeSeries();
-			cpuLoadChart.addTimeSeries(cpuLoadLine, { lineWidth: 1, strokeStyle: 'rgb(180, 180, 180)', fillStyle: OCA.Theming  ? OCA.Theming.color : 'rgb(54, 129, 195)' });
+			cpuLoadChart.addTimeSeries(cpuLoadLine, {
+				lineWidth: 1,
+				strokeStyle: 'rgb(180, 180, 180)',
+				fillStyle: OCA.Theming ? OCA.Theming.color : 'rgb(54, 129, 195)'
+			});
 		}
 
-		$('#cpuFooterInfo').text(t('serverinfo', 'Load average')+": "+cpu1+" ("+t('serverinfo', 'Last minute')+")");
+		$cpu_footer_info.text(t('serverinfo', 'Load average') + ": " + cpu1 + " (" + t('serverinfo', 'Last minute') + ")");
 		cpuLoadLine.append(new Date().getTime(), cpu1);
 	}
 
-	function updateMemoryStatistics (memTotal, memFree, swapTotal, swapFree) {
+	function updateMemoryStatistics(memTotal, memFree, swapTotal, swapFree) {
+		var $mem_footer_info = $('#memFooterInfo');
```

Comment:
Nice nice nice to have ;) We migrate away from jquery step by step and try to avoid jquery for new code. document.getElementById('memFooterInfo) and classList should do the trick. But using jquery is fine.

Reply:
Since there has already been jQuery code present in the JS file I allowed myself to use it.
But i understand that you want to get rid of it.
Probably I will refactor the whole JS file over christmas in a future PR to remove all remains of jQuery (including converting all the AJAX calls)